buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -773,7 +773,6 @@ class BuildPlugin implements Plugin<Project> {
         project.tasks.withType(RandomizedTestingTask) {task ->
             jvm "${project.runtimeJavaHome}/bin/java"
             parallelism System.getProperty('tests.jvms', project.rootProject.ext.defaultParallel)
-            ifNoTests System.getProperty('tests.ifNoTests', 'fail')
```

Comment:
I spent a few minutes wondering if we should change this to `ignore` but I figured that removing it like this seemed right. I couldn't think of a good reason why someone might want to change this on the command line rather than rely on `TestingConventionsTask`.